test/unit/flow_presenter_test.rb

```diff
@@ -8,7 +8,15 @@ def flow_registry
   setup do
     @flow = SmartAnswer::Flow.new do
       name "flow-name"
-      value_question :first_question_key
+      value_question :first_question_key do
+        next_node { question :second_question_key }
+      end
+      value_question :second_question_key do
+        next_node { question :third_question_key }
+      end
+      value_question :third_question_key do
+        next_node { outcome :outcome_key }
+      end
```

Comment:
Does anything make use of this third question? From my quick glance it looked like it might be superfluous.

Reply:
Ah, yeah good spot! I'll remove and fix up.